dev/ci/cases/gfsv17/marine3dvar.yaml

```
@@ -22,5 +22,5 @@ prepoceanobs:
 marineanl:
   SOCA_INPUT_FIX_DIR: {{ HOMEgfs }}/fix/gdas/soca/1440x1080x75/soca
   SOCA_ANL_GEOM: {{ HOMEgfs }}/fix/gdas/soca/720x540x75/soca
-  SOCA_OBS_LIST: {{ HOMEgfs }}/sorc/gdas.cd/parm/soca/obs/obs_list.yaml.j2
-  SOCA_NINNER: 100
+  SOCA_OBS_LIST: {{ HOMEgfs }}/sorc/gdas.cd/parm/marine/obs/obs_list.yaml.j2
```

```suggestion
  SOCA_OBS_LIST: {{ HOMEgfs }}/parm/gdas/marine/obs/obs_list.yaml.j2
```
This file is linked in HOMEgfs/parm, so it should be referenced from there instead of `sorc/gdas.cd`